### ISSUE TITLE
spedup.mp4 not found 

### ISSUE THREAD

**Issue by ismailbgr** [WyattBlue/auto-editor#88]
output of auto-editor --debug

```
Python Version: 3.9.0 64-bit                                                                                                                                                                                                                                 
Platform: Darwin
FFmpeg path: /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_editor/mac-ffmpeg/bin/ffmpeg
Auto-Editor version 20w41a
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'example.mp4':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: mp42mp41
    creation_time   : 2020-05-16T22:31:32.000000Z
  Duration: 00:00:42.45, start: 0.000000, bitrate: 569 kb/s
    Stream #0:0(eng): Video: h264 (Main) (avc1 / 0x31637661), yuv420p(tv, bt709), 1280x720 [SAR 1:1 DAR 16:9], 240 kb/s, 30 fps, 30 tbr, 30k tbn, 60k tbc (default)
    Metadata:
      creation_time   : 2020-05-16T22:31:32.000000Z
      handler_name    : ?Mainconcept Video Media Handler
      encoder         : AVC Coding
    Stream #0:1(eng): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 317 kb/s (default)
    Metadata:
      creation_time   : 2020-05-16T22:31:33.000000Z
      handler_name    : #Mainconcept MP4 Sound Media Handler
Stream mapping:
  Stream #0:1 -> #0:0 (aac (native) -> pcm_s16le (native))
Press [q] to stop, [?] for help
Output #0, wav, to '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/0.wav':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: mp42mp41
    ISFT            : Lavf58.45.100
    Stream #0:0(eng): Audio: pcm_s16le ([1][0][0][0] / 0x0001), 48000 Hz, stereo, s16, 1536 kb/s (default)
    Metadata:
      creation_time   : 2020-05-16T22:31:33.000000Z
      handler_name    : #Mainconcept MP4 Sound Media Handler
      encoder         : Lavc58.91.100 pcm_s16le
size=    7952kB time=00:00:42.41 bitrate=1536.0kbits/s speed= 934x    
video:0kB audio:7952kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.000958%
  ⏳Creating new audio: [██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████] 100.0% done ETA 03:42 PM              
yPointer: 844800
samples per frame: 1600.0
Expected video length: 528.0
Frames written 527ut file.                                                                                                                                                                                                                                   
['/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_editor/mac-ffmpeg/bin/ffmpeg', '-y', '-i', '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/newAudioFile.wav', '-i', '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/spedup.mp4', '-c:v', 'copy', '-crf', '15', '-preset', 'medium', '-movflags', '+faststart', 'example_ALTERED.mp4', '-hide_banner']

Guessed Channel Layout for Input Stream #0.0 : stereo
Input #0, wav, from '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/newAudioFile.wav':
  Duration: 00:00:17.60, bitrate: 1536 kb/s
    Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 48000 Hz, stereo, s16, 1536 kb/s
/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/spedup.mp4: No such file or directory

['/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/auto_editor/mac-ffmpeg/bin/ffmpeg', '-y', '-i', '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/newAudioFile.wav', '-i', '/var/folders/rf/twfwsmj93s535zb_xtjwcbs40000gn/T/tmp0zcie4r6/spedup.mp4', '-c:v', 'copy', '-crf', '15', '-preset', 'medium', '-movflags', '+faststart', 'example_ALTERED.mp4', '-hide_banner']
Error! The file example_ALTERED.mp4 was not created. 
```

**Comment by WyattBlue** [comment]
I wonder if it's because of Python 3.9.  🤔

**Comment by WyattBlue** [comment]
I had another user with a similar bug that they fixed when downgrading to 3.8 so I think that is the solution.
